lib/facility_factory.rb

```diff
@@ -0,0 +1,25 @@
+class FacilityFactory
+    def initialize
+    end
+
+    def create_ny_facility(dmv_locations)
+        dmv_locations.map do |location|
+          Facility.new(
+            name: location[:office_name],
+            address: "#{location[:street_address_line_1]}, #{location[:city]}, #{location[:state]}, #{location[:zip_code]}",
+            phone: location[:public_phone_number]
+          )
+        end
+    end
+    
+    def create_mo_facility(dmv_locations)
+        dmv_locations.map do |location|
+          Facility.new(
+            name: location[:name],
+            address: "#{location[:address1]}, #{location[:city]}, #{location[:state]}, #{location[:zipcode]}",
+            phone: location[:phone]
+          )
+        end
+    end
+     
```

Comment:
Nice job with this class. Good use of using .map to instantiate new Facility instances. One note, when your initialize method is empty, you don't need to include it at all. :)